homeassistant/components/rflink/binary_sensor.py

```diff
@@ -73,12 +76,17 @@ class RflinkBinarySensor(RflinkDevice, BinarySensorEntity, RestoreEntity):
     """Representation of an Rflink binary sensor."""
 
     def __init__(
-        self, device_id, device_class=None, force_update=False, off_delay=None, **kwargs
-    ):
+        self,
+        device_id,
```

Comment:
```suggestion
        device_id: str,
```
Please annotate all arguments if you add types. I believe it should be `str` here.

Reply:
I didn't add at the time because I wasn't quite sure. @javicalle can you confirm that `device_id` is always a string?

Reply:
Acutally I re-read the code and I'm happy that it's a string 

https://github.com/home-assistant/core/blob/2d0ba4b18577f023eeff9405c6ac69452f501252/homeassistant/components/rflink/binary_sensor.py#L35-37